src/main/java/com/jcabi/github/RtStatus.java

```diff
@@ -40,7 +43,6 @@
  * @version $Id$
  * @since 0.23
  */
-@Immutable
```

Comment:
@caarlos0 The immutability checker thinks that `Optional` is mutable for some reason, and this makes compilation fail.